config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hazelcast-configuration
+data:
+  hazelcast.yaml: |-
+    hazelcast:
+      network:
+        join:
+          multicast:
+            enabled: false
```

Comment:
```suggestion
```

Reply:
could this be the reason of test fail? 🤔

Reply:
Shouldn't be. For hazelcast 4.1 it's working without it. I assume that it's because `kind` is a bit slow. Will try to add an additional step for the action so that there are some logs when it fails

Reply:
Looks like the `rest-api` configuration was using for the `livenessProbe/readnessProbe` in k8s. Will restore it back